messages/events/account.go

```diff
@@ -1,8 +1,37 @@
 package events
 
+import "github.com/dogmatiq/example/messages"
+
 // AccountOpened is an event indicating that a new bank account has been opened.
 type AccountOpened struct {
 	CustomerID  string
 	AccountID   string
 	AccountName string
 }
+
+// AccountCredited is an event indicating that a bank account was credited.
+type AccountCredited struct {
+	TransactionID   string
+	AccountID       string
+	TransactionType messages.TransactionType
+	Amount          int64
+}
+
+// AccountDebited is an event indicating that a bank account was debited.
+type AccountDebited struct {
+	TransactionID   string
+	AccountID       string
+	TransactionType messages.TransactionType
+	Amount          int64
+	ScheduledDate   string
+}
+
+// AccountDebitDeclined is an event indicating that a bank account debit was
+// declined.
+type AccountDebitDeclined struct {
+	TransactionID   string
+	AccountID       string
+	TransactionType messages.TransactionType
+	Amount          int64
```

Comment:
I assume we are still doing cents or whatever is one hundredth part of the currency we choose.

Reply:
Yeah, other commets mention that the Amount is cents. Maybe the field should be named more specifically?

Reply:
Nah, I was just confirming for myself mostly, `Amount` is fine.